### PR TITLE
Added support for report attachment download

### PR DESCRIPTION
**Pull request by crestdatasystems**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
- Added the new `cofense-report-attachment-list` command.
- Added the new `cofense-report-attachment-download` command.
- Updated the Docker image to: *demisto/python3:3.10.13.82467*.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/CofenseTriage/.pack-ignore [hunk 1]
```diff
@@ -12,4 +12,53 @@ ignore=IM111
 
 [known_words]
 cofense
-
+cofens
```
Comment (amshamah419) [inline Packs/CofenseTriage/.pack-ignore:15]: Were these just getting picked up by the secrets detection?

**Review by amshamah419 — APPROVED** [review]
Looks good, just one question about the known_words entries.

File: Packs/CofenseTriage/.pack-ignore (hunk 1 above)
Comment (crestdatasystems) [reply Packs/CofenseTriage/.pack-ignore:15]: It was detected by the doc-review command.

**Comment by crestdatasystems** [comment]
@amshamah419 @kgal-pan Please let us know if anything else is required from our side.

**Pull request merged**